code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -28,6 +28,14 @@
 	. = ..()
 	for(var/datum/mutation/human/HM in dna.mutations)
 		HM.on_move(src, NewLoc)
+
+	if(lying && prob(getBruteLoss()*200/maxHealth))

```

Comment:
this should probably be in carbon or base living movement, since not only humans have blood

Reply:
I don't think anything but carbons have blood (or soft crit) so I'll put it in carbon